install_files/ansible-base/group_vars/securedrop_application_server.yml

```diff
@@ -6,10 +6,11 @@ ip_info:
 
 ### Used by the install_local_deb_pkgs role ###
 local_deb_packages:
+  - "securedrop-keyring-0.1.1+{{ securedrop_app_code_version }}-amd64.deb"
+  - "securedrop-config-0.1.0+{{ securedrop_app_code_version }}-amd64.deb"
   - "securedrop-ossec-agent-2.8.2+{{ securedrop_app_code_version }}-amd64.deb"
   - "{{ securedrop_app_code_deb }}.deb"
   - "ossec-agent-2.8.2-amd64.deb"
-  - "securedrop-keyring-0.1.1+{{ securedrop_app_code_version }}-amd64.deb"
```

Comment:
hah! the tables have turned! not in alphabetical order 😛 !

Reply:
Actually the order here is important, since we're installing via `dpkg -i` there's no smart dependency resolution, so we must install the dependencies first, _then_ the packages that depend on them!

Reply:
...... but but..... dang it